### DISCUSSION TITLE
Why was pagination removed from Stars?

### DISCUSSION THREAD

**Discussion by ROODAY** [community/community#10229] [Other Feature Feedback, Questions, & Ideas]
I remember a while back that when looking at your stars you could see the page number, and jump to an arbitrary page number in your stars. Now, we only have previous/next buttons, which is untenable for those with lots of starred repositories. Can the page number please be brought back, as navigating large lists at only a few items at a time is just frustrating.

**Comment by kissu** [comment]
Curious too given the fact that I have 1300 projects starred.
Guess this could be a way of having it back: https://docs.github.com/en/rest/activity/starring?apiVersion=2022-11-28#list-repositories-starred-by-the-authenticated-user

**Comment by 48Naveenkumar** [comment]
Pagination was removed from Stars to simplify the user experience and improve performance. The new infinite scrolling design is meant to offer a smoother and more modern way to browse starred repositories.

**Reply by kissu** [reply]
I don't see any infinite scroll + I don't see how it improves performance either.
If I want to go through all of my 1300 starred projects and order them properly into Lists, I have no ways of doing it nowadays over X days (cuz my browser will just lose that buffer in memory and reset to the first starred project at some point).

Just a loss of basic feature.
Imagine if all the paginated stuff on this platform used that ✨ infinite-smooth-fancy-scroll-UX ✨ flow. 😂

**Reply by ROODAY** [reply]
Why can't we have both? The problem with the infinite scroll is that it uses human unreadable IDs to refer to pages so it's impossible for us to jump across many repos at once. If we had infinite scroll but still had the human readable page=xx in the URL then both the "modern way" of browsing and the way some users have expected to browse repos can work at the same time.

**Reply by kissu** [reply]
Also why do we still have pagination in issues? Isn't it old clunky and suboptimal performance wise? 🫠
